SIL.Windows.Forms/Miscellaneous/SILAboutBox.cs

```diff
@@ -146,7 +156,7 @@ private string GetBuiltOnDate()
 
 			// Use UTC for calculation of build-on-date so that we get the same date regardless
 			// of timezone setting.
-			return string.Format("Built on {0}", fi.CreationTimeUtc.ToString("dd-MMM-yyyy"));
+			return string.Format(LocalizationManager.GetString("AboutDialog.BuiltOnDate", "Built on {0}"), fi.CreationTimeUtc.ToString("dd-MMM-yyyy"));
```

Comment:
I note in passing that this is bogus if your installer (like Bloom's) sets the dates to the install date. At least once that has really tripped us up.